include/aws/cal/private/symmetric_cipher_priv.h

```diff
@@ -31,7 +31,10 @@ struct aws_symmetric_cipher {
     struct aws_byte_buf tag;
     size_t block_size;
     size_t key_length_bits;
+    // deprecated for use, only for backwards compat
+    // use state to represent current state of cipher.
```

Comment:
suggestion: add description for what good used to represent for future, i.e. that it is essentially state == ready